policies/policies/romeo_2082.yml

```diff
@@ -22,31 +22,31 @@ child-policies: []
 flag-romeo:
 
 # Peer review policy url (valid url)
-peer-review-url:
+peer-review-url: http://dev.biologists.org/content/journal-policies
 
 # Does the journal publish the content of peer reviews? (mandatory/optional/no)
-open-reports:
+open-reports: no
 
 # Are reviewer identities revealed to the author? (mandatory/optional/no)
-identities-revealed:
+identities-revealed: optional
 
 # Are reviewer identities published? (mandatory/optional/no)
-identities-published:
+identities-published: no
 
 # Co-reviewer policy url (valid url)
-co-review-url:
+co-review-url: http://dev.biologists.org/content/reviewer-guide#info
 
 # Journal’s policy about co-reviewers - ie people who collaborate with an invited reviewer (free text)
-co-review-policy:
+co-review-policy: We simply ask that: the name of the co-reviewer is reported to the Editor (a field is provided in the report form for this purpose); the same rules of confidentiality and conflict of interest be applied; there is a genuine mentoring process; the senior invited reviewer takes responsibility for the report delivered to the journal.
 
 # Does the journal make it clear in the reviewer invitation email that co-reviewers can contribute? (yes/no)
-co-review-invited:
+co-review-invited: 
```

Comment:
If editing this file, you might as well delete these spaces, so we don't touch lines unless we're changing them.